build-tests/eslint-bulk-suppressions-test/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
This file should not be needed, since monorepo has a root-level `.gitignore` file

Reply:
undid this, thank you Pete